app/src/main/java/com/codesoom/assignment/dto/UserLoginData.java

```diff
@@ -0,0 +1,14 @@
+package com.codesoom.assignment.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+
+@Getter
+@Builder
+@AllArgsConstructor
+public class UserLoginData {
+    private String email;
+
+    private String password;
+}
```

Comment:
이 클래스는 `String` 타입을 두 개 사용하고 있어요. 이 클래스에서는 `@AllArgsConstructor`를 쓰지 않는 것이 좋습니다.
```suggestion
public class UserLoginData {
    private String email;

    private String password;
}
```